src/workers/sqlalchemy_consumer.py

```
@@ -0,0 +1,46 @@
+                try:
+                    self._table_manager.insert(
+                        data=data,
+                        insert_query_template=self._insert_query_template,
+                        conn=conn,
+                        cursor=cursor
+                    )
+                except Exception as e:
+                    self.stop_all_workers()
+                    cursor.close()
+                    conn.close()
+                    raise Exception(e)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)